src/theory/bv/rewrites

```diff
@@ -439,8 +439,37 @@
 (define-rule bv-bitwise-idemp-1 ((x ?BitVec)) (bvand x x) x)
 (define-rule bv-bitwise-idemp-2 ((x ?BitVec)) (bvor x x) x)
 
+(define-rule bv-and-zero-1 ((x ?BitVec))
+  (bvand x (bv 0 (bvsize x))) (bv 0 (bvsize x)))
+(define-rule bv-and-zero-2 ((x ?BitVec))
+  (bvand (bv 0 (bvsize x)) x) (bv 0 (bvsize x)))
+(define-cond-rule bv-and-one-1 ((x ?BitVec) (y ?BitVec))
+  (= y (bvnot (bv 0 (bvsize x))))
+  (bvand x y) x)
+(define-cond-rule bv-and-one-2 ((x ?BitVec) (y ?BitVec))
+  (= y (bvnot (bv 0 (bvsize x))))
+  (bvand y x) x)
+
+(define-cond-rule bv-or-one-1 ((x ?BitVec) (y ?BitVec))
+  (= y (bvnot (bv 0 (bvsize x))))
+  (bvor x y) (bvnot (bv 0 (bvsize x))))
```

Comment:
One nitpicky comment. 

If you split up rules over several lines, it seems easier to read to me when you also add a new line after the match and not just after the conditions even if match and target fit in the same line. It is also done like this in the string rewrite file (https://github.com/cvc5/cvc5/blob/proof-new/src/theory/strings/rewrites)

Reply:
ok I've split the match/target expressions for multi-line rules

Reply:
Thanks